src/views/Home.vue

```diff
@@ -1,36 +1,55 @@
 <template>
   <header
     id="home"
-    :style="{ 'background-image': headerImage }"
+    class="cover-image cover nested-img vh-100 bg-center center"
   >
-    <div class="logo">
+    <NavBar />
+    <div class="pt8 ma3 center measure-wide">
       <img
-        :src="headerLogo"
+        src="../assets/vector/bytie-gradient.svg"
         alt="BYTIE illustration logo"
+        class="img "
       >
-      <h1>ILLUSTRATIONS</h1>
+      <h1 class="near-white center tc tracked-wide ttu f3 ph4">
+        Illustrations
+      </h1>
     </div>
-    <HelloWorld />
+    <!--    <HelloWorld />-->
   </header>
 </template>
 
 <script>
 // @ is an alias to /src
-import HelloWorld from '@/components/HelloWorld.vue'
+/* import HelloWorld from '@/components/HelloWorld.vue' */
+import NavBar from '../components/NavBar'
 /*
 import SomeOtherComponent from '@/components/SomeOtherComponent.vue'
 */
 
 export default {
   name: 'Home',
   components: {
-    HelloWorld /* , SomeOtherComponent */
+    NavBar
+    /* HelloWorld */ /* , SomeOtherComponent */
   },
   data () {
     return {
-      headerImage: './img/header.jpg',
-      headerLogo: './img/vector/default-monochrome-white.svg'
+      headerImage: '../assets/img/header.jpg',
+      headerLogo: '../assets/img/vector/bytie-gradient.svg'
+      // why don't these display?
     }
   }
 }
 </script>
+
+<style scoped>
+.cover-image{
+  background-image: url('../assets/header.jpg');
+}
+.tracked-wide{
+  letter-spacing: 1.05em;
+}
+.pt8{
+  padding-top: 15%;
+}
```

Comment:
instead of using `.pt8` you can use a class or id that better describes your element

You are allowed to do this kind of changes, however they might slow your work on the long run. 

As of the `15%`   it might break the responsiveness of the site